salt/orchestrate/aws/mitx_qa.sls

```diff
@@ -211,6 +211,26 @@ create_mitx_consul_security_group:
     - tags:
         Name: consul-mitx-qa
 
+create_mitx_consul_security_group:
```

Comment:
Can we call this - create_mitx_consul_agent_security_group? It's a bit misleading (until u see the name) given that we already have a function named the same right before it.